### PR TITLE
configuration_manual: Add note about extra fields accepting empty string values

### PR DESCRIPTION
**Pull request by mbettini-ox**

### CODE REVIEW

File: source/configuration_manual/authentication/user_database_extra_fields.rst [hunk 1]
```diff
@@ -22,6 +22,10 @@ fields'. Possibilities are:
   for returning a variable number of fields (e.g. SQL).
 * ``uidgid_file``: Get uid and gid for user based on the given filename.
 
+.. versionchanged:: v3.0.0;v2.4.0
+   Extra fields can now also be set to empty string;
```
Comment (sirainen) [inline source/configuration_manual/authentication/user_database_extra_fields.rst:26]: ..to empty string, while previously they were changed to "yes".

File: source/configuration_manual/authentication/user_database_extra_fields.rst [hunk 2]
```diff
@@ -22,6 +22,10 @@ fields'. Possibilities are:
...
+   extra fields without value will default to empty string as well.
```
Comment (sirainen) [inline source/configuration_manual/authentication/user_database_extra_fields.rst:27]: Is this true? I thought the code still uses "yes" for these on the auth process side? Also it's not very clear what "without value" means, maybe "without '='"

File: source/configuration_manual/authentication/user_database_extra_fields.rst [hunk 3]
```diff
@@ -22,6 +22,10 @@ fields'. Possibilities are:
   for returning a variable number of fields (e.g. SQL).
 * ``uidgid_file``: Get uid and gid for user based on the given filename.
 
+.. versionchanged:: v3.0.0;v2.4.0
```
Comment (sirainen) [inline source/configuration_manual/authentication/user_database_extra_fields.rst:25]: Should also write the same for passdb page.

**Review by sirainen — CHANGES_REQUESTED** [review]

File: source/configuration_manual/authentication/user_database_extra_fields.rst [hunk 4]
```diff
@@ -22,6 +22,11 @@ fields'. Possibilities are:
...
+   Extra fields can now also be set to empty string, while previously they were
+   changed to `yes`. Extra fields without value (without `=`) will default to
+   `yes`.
```
Comment (cmouse) [inline source/configuration_manual/authentication/user_database_extra_fields.rst:28]: you need to use 

```
``yes``
```

**Review by cmouse — APPROVED** [review]

**Pull request merged**